### PR TITLE
colw/include vscode config

### PR DESCRIPTION
**Pull request by colw**
Include vscode config

Thank you! 🚀

---

For contributor:

- [ ] Added changes entries. Run `yarn changelog` for a guided process.
- [ ] Reviewed `Files changed` in the github PR explorer
- [ ] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI

### CODE REVIEW

**Comment by jbibla** [comment]
where is this _config_ you speak of?

**Comment by colw** [comment]
> where is this _config_ you speak of?

Hm. Neat little oversight that was. 😅

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "eslint.validate": [
+    "javascript",
+    "javascriptreact",
```
Comment (jbibla) [inline .vscode/settings.json]: what is `javascriptreact`?

Comment (colw) [reply .vscode/settings.json]: It's copy pasted from here https://eslint.vuejs.org/user-guide/#editor-integrations

Comment (colw) [reply .vscode/settings.json]: Long answer: I don't know, I'm just going off their recommendation.

File: changes/colw_include-vscode-config [hunk 2]
```diff
@@ -0,0 +1 @@
+[Added] [#2807](https://github.com/cosmos/lunie/pull/2807) Include visual studio code config @colw
```
Comment (faboweb) [inline changes/colw_include-vscode-config]: For future reference: I would have used the "repository and tools" type

Comment (colw) [reply changes/colw_include-vscode-config]: Yep, good point. I got this right in the other repo just before.

**Comment by faboweb** [comment]
This doesn't really make a difference for me. Can someone confirm that this has an effect?

**Comment by colw** [comment]
This is hard for me to work with, because mine works fine as is.

What is the desired outcome? Or rather what problem are you having?

**Comment by faboweb** [comment]
@jbibla

**Comment by faboweb** [comment]
Merging anyway because of the browserlist

**Pull request merged**